.eslintrc.yml

```diff
@@ -44,7 +44,6 @@ rules:
   no-multi-str: "off"
   no-unused-vars: "off"
   "@typescript-eslint/no-unused-vars": ["error", { "ignoreRestSiblings": true }]
-  "@typescript-eslint/explicit-module-boundary-types": "error"
```

Comment:
Why remove it? It ensures the result in correct type, isn't it?

Reply:
yes, but as i told you, i didn't migrate all the type yet => next PR